examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    itineraries = get_lib().read_csv('./data/train_set.csv', parse_dates=['checkin'])
</pre><p>shall we keep the dataframe name as <code>train</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>change made</p>